spec/requests/categories_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CategoriesController do
+  fab!(:category) { Fabricate(:category) }
+  fab!(:topic) { Fabricate(:topic, category: category) }
+  fab!(:admin) { Fabricate(:user, admin: true) }
+
+  before do
+    SiteSetting.voting_enabled = true
```

Comment:
Btw is this needed?

Reply:
Indeed yes. Category.can_vote? will be `false` if the plugin is disabled.